notification/docs/DevelopmentGuide.md

```diff
@@ -0,0 +1,44 @@
+# Development guide
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Contents**
+
+- [Requirements](#requirements)
+- [Run Module](#run-module)
+- [Run Tests](#run-tests)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Requirements
+Node.js version 8 LTS or higher is supported.
+
+## Run Module
+Execute `npm сi` to download all dependencies.
+
+In order to run the notification module, you have to provide following environmental variables:
+
+Name | Content | Required | Default value
+------------ | ------------- | ------------- | -------------
+CTP_PROJECT_KEY | commercetools project key (you can get in the [commercetools Merchant Center](https://mc.commercetools.com)) | **YES** |
+CTP_CLIENT_ID | commercetools client ID (you can get in the [commercetools Merchant Center](https://mc.commercetools.com)) | **YES** |
+CTP_CLIENT_SECRET | commercetools client secret (you can get in the [commercetools Merchant Center](https://mc.commercetools.com)) | **YES** |
+LOG_LEVEL | bunyan log level (`trace`, `debug`, `info`, `warn`, `error`, `fatal`)| NO | `info`
+PORT | port on which the application will run | NO | 443
+
+Don't forget to provide all required environment variables
+After setting all variables, execute command `npm run start` to run the module.
+
+## Run Tests
+
+1. Execute `npm run unit-test` to run [Unit tests](../test/unit)
```

Comment:
I would prefer bulleted list instead of numbered list.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/45/commits/5e0c2596cdfc9dc67a161b21035c9f89a800cf23